app/insomnia/enterprise-account-management.md

```diff
@@ -42,19 +42,27 @@ Once the domain is verified, you can configure its settings to control users wit
 
 ### Domain lock
 
-Domain lock allows you to disable access to your Enterprise account for:
-* Existing Hobby users in the domain
-* Uninvited users in the domain
+Use domain lock to remove access to your Insomnia Enterprise account for existing hobby users and uninvited new users.
 
-To enable domain lock, go to your domain settings and click the toggle under **Lock**.
+To enable domain lock:
+- Navigate to your domain settings and click the toggle under **Lock**.
 
-### Invite control
+When you enable domain lock on a specific domain, all users from that domain will no longer be able to access your organization's Insomnia Enterprise account. For example:
+- Kong configures domain lock on @oldkong.com: Ariel is an existing hobby user and wasn’t invited to the Enterprise account. Now, when Ariel@oldkong.com attempts to sign in to Insomnia with that address, she won't have access to the Enterprise account.
+- Kong configures domain lock on @DomainLockExample.com: George doesn't have an Insomnia account, but has an email address george@DomainLockExample.com. When George creates an Insomnia account, the domain lock blocks his sign-in, which means that he can't access the Insomnia Enterprise account or its features.
```

Comment:
```suggestion
- **Existing users without an Enterprise invite:** Ariel is an existing hobby user and wasn’t invited to the Enterprise account. Now, when `Ariel@oldkong.com` attempts to sign in to Insomnia with that address, she won't have access to the Enterprise account.
- **Users without an account, but with a matching email domain:** George doesn't have an Insomnia account, but has an email address `george@DomainLockExample.com`. When George creates an Insomnia account, the domain lock blocks his sign-in, which means that he can't access the Insomnia Enterprise account or its features.

```
Reframed this so it's about what's happening vs. the different domains. Also, the space at the end should get the info note to render correctly.